pcdsdevices/ek9000.py

```diff
@@ -0,0 +1,39 @@
+"""
+Classes for cards attached to EK9000 bus couplers.
+"""
+
+from ophyd import Component as Cpt
+from ophyd import Device
+from ophyd.signal import EpicsSignal, EpicsSignalRO
+
+
+class EL3174_AI_Ch(Device):
+    """
+    EL3174 analog input card channel.
+
+    Can be used to digitize a variety of sensors.
+
+    Parameters
+    ----------
+    prefix : str
+        The PV base of the card.
+    """
```

Comment:
The PVs here look very generic. Is this a more general class than just for an EL3174? Could it potentially be for the whole EL3XXX series?

Reply:
It probably does work for the whole EL3XXX series, but I only have an EL3174 to test with. :)

I can rename this to something more generic.

Reply:
Makes sense! We can leave it as-is and modify later if we find that it is generic.